locales/ja.json

```diff
@@ -5,87 +5,87 @@
         "creatorA": "このサイトは私、LaShawn Toyodaによって作成され、維持されています。サイトの使用に問題がある場合、または誤った情報や古い情報に遭遇した場合は、Twitter {0}でご連絡ください。",
         "creatorQ": "このウェブサイトを作成したのは誰ですか？",
         "header": {
-            "headline": "About Find A Doc, the health database for Japan's international community"
+            "headline": "Find a Doc、 日本インターナショナルコミュニティーの健康データベースについて"
         },
         "kofi": {
-            "action": "Donate a cup",
-            "headline": "Buy us coffee",
+            "action": "一杯を寄付する",
+            "headline": "コーヒーを寄付する",
             "text": {
-                "0": "We are running this database after our 9-5 jobs. Help us stay awake!",
-                "1": "Consider supporting us if this project has helped you or someone you know."
+                "0": "私たちは、通常の仕事の後に、このデータベースを動かしています。眠らないように助けてください！",
```

Comment:
1. Try to avoid for the most part using first-person pronouns except where appropriate. It doesn't feel natural in Japanese to do this. Better something like: 「このデーターベースは通常勤務と別にボランティアによって開発されたものです」
2. I don't think the idiom of 'help us stay awake' really works in Japanese the same way. Try something that speaks just directly to the action like 「支援していただけませんか？」

Reply:
All right. Feel a bit sad about losing "Help us stay awake!" nuance ww

Reply:
Yeah, it's pretty sweet in the English version =). It doesn't really carry over in the way we'd want it it to tho.